src/apps/VirtualApp/VirtualChat.js

```diff
@@ -48,7 +48,7 @@ class VirtualChat extends Component {
       },
       error: (err) => {
         console.error("  -- Error join room", err);
-        captureException(err, {source: "Textroom", err: `Error joining room (${roomid}): ${err}`});
+        captureMessage(`Chatroom error: join room (${roomid}) - ${err}`, {source: "Textroom", err}, 'error');
```

Comment:
Like it is message and not exception due to the fact that it is an acceptable behavior?

Why err here is an object and not string? Should it always be an object?